app/gateway/changelog.md

```diff
@@ -5,6 +5,22 @@ no_version: true
 
 <!-- vale off -->
 
+## 3.2.2.0
```

Comment:
Do we also want to mention something about the yum install that Tyler mentioned? I was thinking maybe adding the workaround and reason for why it errors?

> [KAG-892](https://konghq.atlassian.net/browse/KAG-892) is still an issue so customers who try to yum install this will get a Public key for kong-enterprise-edition-3.2.1.0.rhel7.amd64.rpm is not installed error. They can get around this error by manually downloading the package from [downloads.konghq.com](http://downloads.konghq.com/) and installing it. The package is signed, it is just signed with a different (rotated) key from the metadata service, and yum doesn't like that.

Reply:
yes, is missing from the changelog but agree it's a known issue w/ workaround.